Test/Mock/MockTemplating.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Test\Mock;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Mocking class for template usage.
+ *
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ */
+class MockTemplating implements EngineInterface
```

Comment:
We can't make it final now, because the old `FakeTemplating` class is extending this class for BC.